web/islands/RemoteFollowModal.tsx

```
@@ -0,0 +1,294 @@
+                <div class="flex items-start gap-3">
+                  {actorInfo.value.icon && (
+                    <img
+                      src={actorInfo.value.icon}
+                      alt={t("remoteFollow.profileImageAlt")}
+                      class="w-10 h-10 rounded-full flex-shrink-0"
+                    />
+                  )}
+                  <div class="flex-1">
+                    <h4
+                      class="font-medium text-gray-900 dark:text-gray-100"
+                      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)